### PR TITLE
Add two English articles and short descriptions for Open Source projects

### PR DESCRIPTION
**Pull request by Sherlock113**

### CODE REVIEW

File: content/conference/en/logging.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+![](https://pek3b.qingstor.com/kubesphere-docs/png/20191001091801.png)
+Loki Architecture (Source: official blog of Grafana Labs)
+
+With Loki, the data of Index and chunks are stored in different media as shown below. The Index can be stored in Cassandra and BoltDB while the chunks can be saved in a local disk, through cloud object storage or Minio that supports the S3 protocol. That means a large number of log data can be stored at a low cost on the one hand and the need of users for rapid log search can be met on the other.
+
+![](https://pek3b.qingstor.com/kubesphere-docs/png/20191001091838.png)
+
+Loki Storage (Source: official blog of Grafana Labs)
+
+As a logging solution that answers the call of the cloud native times, Loki has quickly come to the spotlight as it features low cost, great scalability and high availability, and close integration with Kubernetes and Prometheus. It is expected to be on a par with Prometheus as the preferred solution to cloud native log management.
```
Comment (FeynmanZhou) [inline content/conference/en/logging.md:103]: a `par` with Prometheus => a `part` with Prometheus?

Comment (Sherlock113) [reply content/conference/en/logging.md:103]: No, "be on a par with" is a set phrase, meaning "as good as" or "equal in importance or quality to".

Comment (FeynmanZhou) [reply content/conference/en/logging.md:103]: Got it, the first I see this phrase👍

File: content/conference/en/porter.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+title: 'Porter: An Open Source Load Balancer for Kubernetes in a Bare Metal Environment'
+type: 'KubeCon'
+author: 'Song Xuetao'
+createTime: '2019-06-24'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-docs/png/20190930095713.png'
+---
+
+We know that we can use the service of LoadBalancer in the Kubernetes cluster to expose backend workloads externally. Cloud providers often offer cloud LB plugins, which requires the cluster to be deployed on a specific IaaS platform. However, many enterprise users often deploy the Kubernetes cluster in a bare machine, especially when it is used for the production environment. For the local bare machine cluster, Kubernetes does not provide LB implementation. Porter is an open source load balancer designed specifically for the bare metal Kubernetes cluster, which serves as an excellent solution to this problem.
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: bare machine => bare metal

Comment (Sherlock113) [reply content/conference/en/porter.md]: Ok, noted

File: content/conference/en/porter.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+Initially, NodePort is not designed for the exposure of services in the production environment which is why large port numbers are used by default.
+### LoadBalancer
+LoadBalancer is a preferred solution by Kubernetes to service exposure. However, this cannot be done without the load balancer offered by cloud providers, which means the Kubernetes cluster has to be deployed in the cloud. Here is how LoadBalancer works:
+
+![图片](https://uploader.shimo.im/f/CjKPrUhvUCYJVWby.png!thumbnail)
+
+The LoadBalancer service is achieved through the LB plugin offered by cloud providers. The package Kubernetes.io/cloud-provider will choose the appropriate backend service and expose it to the LB plugin, which creates a load balancer accordingly. That means network traffic will be distributed in the cloud service, avoiding a single point of failure and performance bottlenecks that may occur in NodePort. As mentioned above, LoadBalancer is a preferred solution by Kubernetes to service exposure, but it is only limited to the Kubernetes service offered by cloud providers. For the Kubernetes cluster that is deployed physically or in a non-cloud environment, this approach may not be applicable.
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: deployed physically => deployed in a bare metal environment

File: content/conference/en/porter.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+## Porter Introduction
+
+Porter is an open source cloud native load balancing plugin designed by the KubeSphere development team based on Border Gateway Protocol (BGP). It meanly features:
+
+1. ECMP routing load balancing
+2. BGP dynamic routing configuration
+3. VIP management
+
+All Porter codes are open source and documents are available in [GitHub](https://github.com/kubesphere/porter). You are welcome to star and use it.
+## Rapid Deployment
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: Rapid Deployment => How to install Porter

File: content/conference/en/porter.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+- [Test in the QingCloud Platform Using a Simulated Router](https://github.com/kubesphere/porter/blob/master/doc/zh/simulate_with_bird.md)
+
+## Principle
+
+### ECMP
+
+Equal-Cost Multi-Path (ECMP) means the package forwarding to a same destination can occur along multiple paths of equal cost. When the device supports ECMP, the three-layer traffic that is sent to the target IP or network segment can be distributed by different paths, achieving network load balancing. Besides, once a certain path malfunctions, other paths can finish the forwarding process instead,  serving as the routing redundant backup. Please refer to the image below:
+![图片](https://uploader.shimo.im/f/FfwaMcNCNY8glOhH.png!thumbnail)
+
+With the help of the virtual router, ECMP can select the next hop (Pod) according to Hash functions from the existing routing paths for a certain IP (the corresponding VIP of the service). This is how load balancing is achieved. As virtual routers support ECMP in general, Porter only needs to check the Kubernetes API server and deliver the corresponding information of backend Pod of a service to the router.
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: Hash functions => Hash algorithm

File: content/conference/en/porter.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+Agent is a lightweight component to monitor VIP resources and add Iptables rules for external access to the VIP. By default, the kernel Forward table will drop any external access to VIP.
+
+## Designed for Cloud Natives
+
+All resources in Porter are CRD, including VIP, BGPPeer and BGPConfig. Users who are used to Kubectl will find Porter very easy to use. For advanced users who want to customize Porter, Kubernetes API can be called directly for tailor-made development. The kernel controller of Porter will soon support high availability (HA).
+
+## Cautions
+
+The VIP traffic of user access will go to a node in the Kubernetes cluster under BGP. This is because the routes advertised by Porter are also nodes instead of podip which is inaccessible externally. The path from a node to a pod is maintained by kube-proxy as below:
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: podip => Pod IP

File: content/conference/en/porter.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+## Architecture
+
+![图片](https://uploader.shimo.im/f/u8DjFjHZ2sU1aKSY.png!thumbnail)
+
+Porter has two components: a kernel controller and an agent deployed on each node. The main functions of the controller include:
```
Comment (FeynmanZhou) [inline content/conference/en/porter.md]: kernel => core

File: src/locales/zh-CN/common.json [hunk 8]
```diff
@@ -112,17 +112,17 @@
...
+  "OpenPitrix is an open source multi-cloud application management platform. It is useful in packing, deploying and managing applications of different kinds (e.g. traditional, microservice and serverless) in multiple cloud platforms, including AWS, Kubernetes, QingCloud and VMWare.": "开源的多云应用程序管理平台，用来在多云环境下打包、部署和管理不同类型的应用，包括传统应用、微服务应用以及 Serverless 应用等，其中云平台包括 AWS、Kubernetes、QingCloud、OpenStack、VMWare",
+  "It is a load balancing plugin for the physical deployment of Kubernetes, made possible by a physical switch. On the back of BGP and ECMP, the best performance can be achieved with high availability, providing users with a consistent experience as they can enjoy both in a LoadBalancer service exposed in a physical environment and in the cloud.": "适用于物理部署 Kubernetes 环境的负载均衡器插件，Porter 使用物理交换机实现，利用 BGP 和 ECMP从而达到性能最优和高可用性，提供用户在物理环境暴露 LoadBalancer 类型服务与云上获得一致性体验。",
```
Comment (FeynmanZhou) [inline src/locales/zh-CN/common.json]: It is a load balancing plugin for the physical deployment of Kubernetes, => It is a bare metal load balancer for Kubernetes cluster,

Comment (FeynmanZhou) [inline src/locales/zh-CN/common.json]: made possible by a physical switch => Porter is implemented by using the physical switch

Comment (FeynmanZhou) [inline src/locales/zh-CN/common.json]: providing users with a consistent experience as they can enjoy both in a LoadBalancer service exposed in a physical environment and in the cloud.
=> allowing you to create Kubernetes services of type LoadBalancer in the bare metal cluster, which makes you enjoy the consistent experience with the cloud.

**Comment by FeynmanZhou** [comment]
@Sherlock113 You can modify these files according to my comments, and push a new commit when you are done.

**Pull request closed without merging**